web-client/src/App.tsx

```diff
@@ -1,27 +1,21 @@
 import React, { ReactElement } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Provider } from 'react-redux';
+import { enableAllPlugins } from "immer"
+import ExampleContainer from './containers/exampleContainer';
+import configureStore from './store';
+
+// Later we can check if we need all immer plugins
+enableAllPlugins()
 
 function App(): ReactElement {
+  const store = configureStore()
```

Comment:
if ';' is not mandatory, we should add the rule to enforce it, or the other way around. Personally prefer to have it than not

Reply:
Personally i also agree with @comoser

Reply:
Same as https://github.com/reach4help/reach4help/pull/68#discussion_r396341297